index.html

```diff
@@ -2175,58 +2175,56 @@ <h4 id=el-figcaption>`figcaption`</h4>
     <tr>
       <th>[[wai-aria-1.2]]</th>
       <td>
-        No corresponding role
+        <a class="core-mapping" href="#role-map-caption">`caption`</a> role
       </td>
     </tr>
     <tr>
       <th><a href="https://msdn.microsoft.com/en-us/library/dd373608%28v=VS.85%29.aspx">MSAA</a> + <a href="http://accessibility.linuxfoundation.org/a11yspecs/ia2/docs/html/">IAccessible2</a></th>
       <td>
         <div class="role">
-          <span class="type">Roles:</span> `ROLE_SYSTEM_TEXT`; `IA2_ROLE_CAPTION`
+          <span class="type">Role:</span> Use WAI-ARIA mapping
         </div>
         <div class="relations">
-          <span class="type">Relations:</span> `IA2_RELATION_LABEL_FOR` with parent <a href="#el-figure">`figure`</a> element
+          <span class="type">Relations:</span>
+          If the first instance of a child `figcaption` to its parent <a href="#el-figure">`figure`</a>, then `IA2_RELATION_DETAILS_FOR` with parent `figure`. 
         </div>
       </td>
     </tr>
     <tr>
       <th><a href="https://msdn.microsoft.com/en-us/library/ms726297%28v=VS.85%29.aspx">UIA</a></th>
       <td>
         <div class="ctrltype">
-          <span class="type">Control Type:</span> `Text`
+          <span class="type">Role:</span> Use WAI-ARIA mapping
         </div>
       </td>
     </tr>
     <tr>
       <th><a href="https://gnome.pages.gitlab.gnome.org/atk/">ATK</a></th>
       <td>
         <div class="role">
-          <span class="type">Role:</span> `ATK_ROLE_CAPTION`
+          <span class="type">Role:</span> Use WAI-ARIA mapping
         </div>
         <div class="relations">
           <span class="type">Relations:</span>
-          `ATK_RELATION_LABEL_FOR` with parent <a href="#el-figure">`figure`</a> element
+          If the first instance of a child `figcaption` to its parent <a href="#el-figure">`figure`</a>, then `RELATION_DETAILS_FOR` with parent `figure` element.
         </div>
       </td>
     </tr>
     <tr>
       <th><a href="https://developer.apple.com/reference/appkit/nsaccessibility">AX</a></th>
       <td>
         <div class="role">
-          <span class="type">AXRole:</span> `AXGroup`
-        </div>
-        <div class="subrole">
-          <span class="type">AXSubrole:</span> `(nil)`
-        </div>
-        <div class="roledesc">
-          <span class="type">AXRoleDescription:</span> `"group"`
+          <span class="type">Role:</span> Use WAI-ARIA mapping
         </div>
       </td>
     </tr>
     <!-- <th><a href="#accessible-name-and-description-computation">Naming Algorithm</a></th> -->
     <tr>
       <th>Comments</th>
       <td>
+        <p>
+          Only the first child instance of a <a href="#el-figcaption">`figcaption`</a> this is present in the accessibility tree
+          provides details for its parent `figure`.</p>
```

Comment:
Same caveat as above. Please ensure my editorial changes don't break your intended meaning.

```suggestion
        <p>Only the first descendant <a href="#el-figcaption">`figcaption`</a> element in a `figure` element provides details for its ancestor `figure` element.</p>
```

Reply:
Aside from the editorial changes, why would only the first be related programmatically? Is this working around a bug in the API? If so, should that be filed?

Reply:
it's invalid HTML to have more than one figcaption within a figure, unless subsequent figcaptions are within their own nested figures. the figcaption must be a direct child of the figure.    i can see an argument to go beyond what html says is valid to provide more info for AT, but if we do that i'd want it to be in another PR.

so going to make some changes based on your editorial suggestions - but keeping references to 'child' and 'first instance'